src/installing-workflow/experimental-native-ingress.md

```diff
@@ -0,0 +1,80 @@
+# Experimental Native Ingress
+
+## Install Deis Workflow (With experimental native ingress support)
+
+Now that Helm is installed and the repository has been added, install Workflow with a native ingress by running:
+
+```
+$ helm install deis/workflow --namespace deis --set global.experimental_native_ingress=true,controller.platform_domain=deis.com
+```
+
+Where `global.hostname` is a **required** parameter that is traditionally not required for Workflow. In this example we are using `deis.com` for `$hostname`.
+
+
+ 
+Helm will install a variety of Kubernetes resources in the `deis` namespace.
+Wait for the pods that Helm launched to be ready. Monitor their status by running:
+
+```
+$ kubectl --namespace=deis get pods
```

Comment:
Kubernetes will have the pod information stored **very** quickly, and I think adding a `-w` adds an implicit requirement on noting `^C` and that sounds like adding a lot for a small gain.